jbmc/src/java_bytecode/java_types.h

```diff
@@ -695,7 +695,7 @@ inline typet java_type_from_string_with_exception(
 }
 
 /// Get the index in the subtypes array for a given component.
-/// \param t The type we search for the subtypes in.
+/// \param gen_types The subtypes array.
```

Comment:
Nit-pick: add a colon after `gen_types` as is recommended in the coding guidelines (but arguably it's not done here for the other parameters).